doc/writing.md

```diff
@@ -16,9 +16,8 @@ features!
 
 If you are new to Markdown, you should give [this 10-minute Markdown
 tutorial](http://commonmark.org/help/tutorial/) a try. Monod supports CommonMark
-(a strongly defined, highly compatible specification of Markdown). Because it
-does not support HTML for security concerns, Monod provides various syntax
-extensions.
+(a strongly defined, highly compatible specification of Markdown). It does not support
```

Comment:
@gar Rightly mentioned. "It" makes it unclear what it is we are referring to. Will make the change.